src/WordPress.php

```diff
@@ -205,6 +227,54 @@ public function create( $args, $assoc_args ) {
 		}
 		$this->site_data['app_sub_type'] = $mu ?? 'wp';
 
+		$vip_wp_content_repo = \EE\Utils\get_flag_value( $assoc_args, 'vip' );
+
+		if ( ! empty( $vip_wp_content_repo ) && is_bool( $vip_wp_content_repo ) ) {
+			$vip_wp_content_repo = $this->vip_go_skeleton;
+		}
+
+		if ( ! empty( $vip_wp_content_repo ) ) {
+			$this->is_vip                    = true;
+			$this->site_meta['vip_repo_url'] = $vip_wp_content_repo;
+
+			if ( $this->vip_go_skeleton === $vip_wp_content_repo ) {
+				\EE::log( 'VIP Skeleton repo will be used for wp-content directory: ' . $vip_wp_content_repo );
+			} else {
+				\EE::warning( 'This repo will be clone at wp-content directory make sure the repo has wp-content data.' );
+				\EE::confirm( 'Are you sure want to continue?' );
+
+				\EE::log( "Checking VIP repo access..." );
+
+				$is_valid_git_url = false;
+
+				if ( 0 === strpos( $vip_wp_content_repo, 'git@github.com' ) ) {
+					$is_valid_git_url = true;
+				}
+
+				if ( 0 === strpos( $vip_wp_content_repo, 'https://github.com' ) ) {
+					$is_valid_git_url = true;
+				}
+
+				if ( empty( $is_valid_git_url ) ) {
+					\EE::error( 'Vip repo url is not valid github repo url. Please add valid ssh/https github url.' );
+				}
+			}
+
+			$git_check = \EE::exec( 'command -v git' );
```

Comment:
This check is actually redundant in case of EasyEngine 4. If we're giving support for `--vip` and know that `git clone` will be done, then we need to add git to the php image; if we add git to the php image, we can be assured that git will be there because the container will always have git installed.

No harm adding this check though.

Reply:
We can't add git to php image since if we do that then user will have to add their ssh key into the container which should not happen and without that ssh key we can't do git clone. So the `git clone` should be done from host machine.